rustls-libssl/src/x509.rs

```diff
@@ -19,12 +20,23 @@ pub struct OwnedX509Stack {
 }
 
 impl OwnedX509Stack {
+    /// Make an empty stack.
     pub fn empty() -> Self {
         Self {
             raw: unsafe { OPENSSL_sk_new_null() as *mut stack_st_X509 },
         }
     }
 
+    pub fn from_rustls(certs: &Vec<CertificateDer<'static>>) -> Self {
+        let mut r = Self::empty();
+        for c in certs {
+            let item = OwnedX509::parse_der(c.as_ref()).unwrap();
```

Comment:
Should this `unwrap()`? It looks like we pass through the DER from the staged certs without doing any verification up-front. I think it might be better to return a `Result` so something higher in the stack can handle bad inputs.

Reply:
It certainly shouldn't!